front-end/src/components/productTable/ProductIcon.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Styles from './Styles';
+
+function ProductIcon(props) {
+  const { classes, product } = props;
+  return (
+    <Grid item>
+      <Paper className={classes.paper} component="div">
+        {/* eslint-disable-next-line no-octal-escape */}
+        <img
```

Comment:
umm, o nėra galimybės išspręsti eslint error'us?

Reply:
Jei istaisau, kita errora meta, tai ne

Reply:
Tada eslint'o konfiguracijoje nustatyk, kad nemestų to error'o, kode tokių snippetų nepaliekam